test/prompts/test_prompt_base.py

```diff
@@ -140,7 +140,7 @@ def test_code_prompt_execute(monkeypatch):
         "a = 1\nprint('Hello, World!')", code_type="python"
     )
     result = code_prompt.execute()
-    assert result == "Hello, World!\n"
+    assert result == "Hello, World!\nNone\n"
```

Comment:
It is a bit confusing why there is a `None` in the output. Can you explain? This may confuse the agent as well.